docs/source/quickstarts/gke.md

```diff
@@ -44,7 +44,7 @@ node-pools create 'scyllaclusters' \
 --zone='us-central1' \
 --cluster='my-k8s-cluster' \
 --node-version="latest" \
---machine-type='n1-standard-16' \
+--machine-type='n2-standard-16' \
 --num-nodes='4' \
```

Comment:
Why are we creating 4 nodes? (and not 3)

Reply:
unrelated to this PR but we always have
https://github.com/scylladb/scylla-operator/blame/de94fed/docs/source/gke.md#L99

I suppose it's the manager cluster or the option to scale up